cmd/ts-info/app/video.go

```diff
@@ -10,12 +10,29 @@ import (
 	"github.com/asticode/go-astits"
 )
 
+type streamStatistics struct {
+	Type       string  `json:"streamType"`
+	Pid        uint16  `json:"pid"`
+	FrameRate  float64 `json:"frameRate"`
+	TimeStamps []int64 `json:"-"`
+	MaxStep    int64   `json:"maxStep,omitempty"`
+	MinStep    int64   `json:"minStep,omitempty"`
+	AvgStep    int64   `json:"avgStep,omitempty"`
+	// RAI-markers
+	RAIPTS         []int64 `json:"-"`
+	IDRPTS         []int64 `json:"-"`
+	RAIGOPDuration int64   `json:"RAIGoPDuration,omitempty"`
+	IDRGOPDuration int64   `json:"IDRGoPDuration,omitempty"`
+	// Errors
+	Errors []string `json:"errors,omitempty"`
+}
+
 type naluFrameData struct {
 	PID   uint16     `json:"pid"`
 	RAI   bool       `json:"rai"`
 	PTS   int64      `json:"pts"`
-	DTS   *int64     `json:"dts,omitempty"`
-	NALUS []naluData `json:"nalus"`
+	DTS   int64      `json:"dts,omitempty"`
+	NALUS []naluData `json:"nalus,omitempty"`
```

Comment:
It shouldn't happen, but I think we should show that there are no NALUs as an empty slice